pkg/test/socialevent/socialevent.go

```diff
@@ -47,3 +47,15 @@ func WithActivationCount(value int) Option {
 		event.Status.ActivationCount = value
 	}
 }
+
+func WithUserTier(tier string) Option {
```

Comment:
Just noticed there's no `WithMaxAttendees` modifier. Are we missing some testing in that area?

Reply:
good point, let me check that

Reply:
added in 302b374